low-level/framework/rabbitmq/logging_processor.py

```diff
@@ -104,7 +113,16 @@ def run(self):
 
         self._log_debug("Start accepting requests")
         try:
-            self._connection.consume(callback=self._process_msg)
+            while True:

```

Comment:
we usually schedule threads with some frequency right, why 'while true' loop here?

Reply:
Current message bus implementing don't have  event/callback based methods.
So, we will have to poll until message is available.

Reply:
Add 0.1 sec delay in every loop. otherwise it will kill the cpu.